README.md

```diff
@@ -384,6 +385,38 @@ Recursively flattens array.
 
 **[⬆ back to top](#quick-links)**
 
+### _.fromPairs
+
+Returns an object composed from key-value pairs.
+
+  ```js
+  // Underscore/Lodash
+  _.fromPairs([['a', 1], ['b', 2]]);
+  // => { 'a': 1, 'b': 2 }
+
+  // Native
+  var fromPairs = function(arr) {
```

Comment:
I think `const` is more better than `var`.

Reply:
@terrierscript  it seems that we use both in our other examples. :smile: 
It could be good to raise a separate PR to fix them all.

Reply:
I did not notice other examples had consts, only vars. I use usually const as a default unless a var is absolutely needed.